chapter_1/networking.py

```diff
@@ -0,0 +1,7 @@
+import socket
+socket.setdefaulttimeout(2)
+s = socket.socket()
+s.connect(("localhost", 21))
```

Comment:
Please update this to port 2121 so we will be able to connect to the FTP server listening on port 2121.

```suggestion
s.connect(("localhost", 2121))
```

Reply:
Done.